build/cqs.tsv

```diff
@@ -1,19 +1,25 @@
+asm:asm	3.1	compile
 asm:asm	3.3.1	compile
 com.101tec:zkclient	0.3	compile
+com.101tec:zkclient	0.7	compile
+com.101tec:zkclient	0.8	compile
```

Comment:
Have an entry for zkclient 0.3.  

I *think* that zkclient is transitive of Kafka.  Since we have been treating Kafka as a 'provided' / workswith / doesn't-ship-dep, its transitives should not be in the build.